.github/workflows/livecheck.yml

```diff
@@ -0,0 +1,185 @@
+        with:
+          path: ${{ steps.set-up-homebrew.outputs.gems-path }}
+          key: ${{ runner.os }}-rubygems-${{ steps.set-up-homebrew.outputs.gems-hash }}
+          restore-keys: ${{ runner.os }}-rubygems-
+
+      - name: Download failed list
+        uses: actions/download-artifact@v4
+        with:
+          name: failed.txt
+
+      - name: Checkout autobump removal branch
+        working-directory: ${{ steps.set-up-homebrew.outputs.repository-path }}
+        run: |
+          if git ls-remote --exit-code --heads origin "${BRANCH}"
+          then
+            git fetch origin "${BRANCH}"
+            git checkout "${BRANCH}"
+            git checkout .github/autobump.txt
+          else
+            git checkout --no-track -B "${BRANCH}" origin/master
+          fi
+
+      - name: Remove failed formulae from autobump list
+        env:
+          HOMEBREW_CORE: ${{ steps.set-up-homebrew.outputs.repository-path }}
+        run: |
+          tempfile="$(mktemp -t autobump.XXXXXX)"
+          autobump_list="${HOMEBREW_CORE}/.github/autobump.txt"
+          comm -23 <(sort -u "${autobump_list}") <(sort -u failed.txt) >"${tempfile}"
+          mv "${tempfile}" "${autobump_list}"
+
+      - name: Commit autobump changes
+        id: commit
+        env:
+          GIT_COMMITTER_NAME: ${{ steps.git-user-config.outputs.name }}
+          GIT_COMMITTER_EMAIL: ${{ steps.git-user-config.outputs.email }}
+          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
+        working-directory: ${{ steps.set-up-homebrew.outputs.repository-path }}
+        run: |
+          if git diff --stat --exit-code .github/autobump.txt
+          then
+            echo "push=false" >>"${GITHUB_OUTPUT}"
+          else
+            git add .github/autobump.txt
+            git commit -m "autobump: remove livecheck failures" \
+                       -m "Autogenerated by the [livecheck](${RUN_URL}) workflow."
+
+            echo "push=true" >>"${GITHUB_OUTPUT}"
+          fi
+
+      - name: Push changes
+        if: fromJSON(steps.commit.outputs.push)
+        uses: Homebrew/actions/git-try-push@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          directory: ${{ steps.set-up-homebrew.outputs.repository-path }}
+          branch: ${{ env.BRANCH }}
+          force: true
+        env:
+          GIT_COMMITTER_NAME: ${{ steps.git-user-config.outputs.name }}
+          GIT_COMMITTER_EMAIL: ${{ steps.git-user-config.outputs.email }}
+          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
+
+      - name: Open pull request
+        if: fromJSON(steps.commit.outputs.push)
+        env:
+          GITHUB_TOKEN: ${{ secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN }}
+        run: |
+          if ! jq --exit-status '.state == "OPEN"'
+          then
+            gh pr create --fill
+          fi < <(gh pr view --json=state)
```

Comment:
`gh pr view` does not work well when there are multiple PRs with the same branch name. (Try to checkout a new branch named `sponsors-maintainers-man-completions` in `brew --repo` and see.) The `sync-shared-config` workflow [relies on the API](https://github.com/Homebrew/.github/blob/c65c8b0c816d261969bfd897caff6bb472f82ff7/.github/workflows/sync-shared-config.yml#L113-L129) for this reason.